docker-compose.yml

```diff
@@ -103,6 +103,32 @@ services:
     volumes:
       - ./build/certs:/etc/certs
       - ./build/mongod.conf:/etc/mongod.conf
+  
+  # used for testing a replica set
+  mongodb_replset:
+    build:
+      context: ./build/deps
+      dockerfile: ${MONGO_DOCKERFILE:-mongo6}.Dockerfile
+    container_name: mongodb_replset
+    command: mongod --port 27017 --bind_ip_all --replSet rs0
```

Comment:
In other container like` mongodb` and `mongodb_secured`, .conf file is used. I suggest we keep the consistency, if we want to add a new docker.

Reply:
Sounds good!

Reply:
Fixed